backstop_data/engine_scripts/puppet/search/find-contact.js

```diff
@@ -5,5 +5,5 @@ const Page = require('../page-objects/crm-page.js');
 module.exports = async (engine, scenario, vp) => {
   const page = await Page.build(engine, scenario, vp);
 
-  await page.clickAndWaitForNavigation('#_qf_Basic_refresh');
+  await page.clickAndWaitForSuccessfulSearch();
```

Comment:
We had different selectors for every `clickAndWaitForNavigation` call. previously. But now all are unified, does that mean `'.crm-form-submit[value="Search"]';` takes care of all these cases?

Reply:
yes, and in case there are exceptions you can still pass a custom selector, see https://github.com/compucorp/backstopjs-config/pull/35/files/11507c560a5df8d8734099020be62700821e00ac..4a76f9d70776d8b04dfaea97d25e8bd54bebd142#diff-500c1fbcdbad84f587a86e8cbc452161R105